RELEASE_TEMPLATE.md

```diff
@@ -37,38 +37,52 @@ Times are relative to the start of the release.
-- [ ] Ensure that the technician has a Docker Hub account and permissions to push into the bloxoss repositories.
+- [ ] Ensure that `deploy/docker/conf/docker-compose.yml` has been updated with the correct image versions in the release-$$blox_version$$ branch.
+- [ ] Ensure that `deploy/aws/conf/cloudformation_template.json` has been updated with the correct image versions in the release-$$blox_version$$ branch.
+- [ ] Ensure that `IamRoleInstance,IamRoleService,IamRoleTask,IamRoleLambda` in `deploy/aws/conf/cloudformation_template.json` have the correct role permissions in the release-$$blox_version$$ branch.
+- [ ] Ensure that a $$blox_version$$ change log entry exists in `CHANGELOG.md` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/README.md` and `daemon-scheduler/README.md` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/VERSION` and `daemon-scheduler/VERSION` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/versioning/version.go` and `daemon-scheduler/versioning/version.go` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the release-$$blox_version$$ branch contains the latest changes in the dev and master branch.
+- [ ] Ensure that the release-$$blox_version$$ branch commit hash is set to $$github_hash$$.
+- [ ] Ensure that the technician has a GitHub account and permissions to merge into the Blox dev and master branches.
+- [ ] Ensure that the technician has a Docker Hub account and permissions to push and remove images in the bloxoss repositories.
 - [ ] Ensure that the technician has logged into Docker Hub via `docker logout` and `docker login`.
+- [ ] Delete all bloxoss local images before building new ones.
+- [ ] Build the bloxoss/cluster-state-service:pre-$$blox_version$$ image and publish to Docker Hub.
+- [ ] Build the bloxoss/daemon-scheduler:pre-$$blox_version$$ image and publish to Docker Hub.
+- [ ] Ensure that the bloxoss/cluster-state-service:pre-$$blox_version$$ image shows the correct version and commit hash.
+- [ ] Ensure that the bloxoss/daemon-scheduler:pre-$$blox_version$$ image shows the correct version and commit hash.
+- [ ] Ensure that the cluster-state-service end-to-end tests pass against the pre-$$blox_version$$ images.
+- [ ] Ensure that the daemon-scheduler end-to-end tests pass against the pre-$$blox_version$$ images.
+- [ ] Ensure that the daemon-scheduler integration tests pass against the pre-$$blox_version$$ images.
 - [ ] Release activities:
-- [ ] Merge the GitHub pull request into the master branch.
+- [ ] Push the release-$$blox_version$$ branch up to the Github dev and master branches.
 - [ ] Create a v$$blox_version$$ release tag in GitHub with commit $$github_hash$$.
```

Comment:
I've updated this to two steps, draft and publish:
* Draft a v$$blox_version$$ release in GitHub against the master branch
* Publish the v$$blox_version$$ release in GitHub